_posts/2020-10-01-introducing-points.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Introducing Points - An blind estimation tool from OmbuLabs"
+date: 2020-10-16 12:00:00
+categories: ["open-source", "agile"]
+author: "fionadl"
```

Comment:
```suggestion
author: ["fionadl", "abizzinotto"]
```

Reply:
@FionaDL should be `authors` (plural) to work